src/app/auth/register/register.component.css

```diff
@@ -0,0 +1,164 @@
+.flex-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  box-sizing: border-box;
+}
+
+.login-container {
+  width: 100%;
+  max-width: 418px;
+  height: 710px;
+  box-sizing: border-box;
+  background-color: white;
+  border-radius: 12px;
+  box-shadow: 0 0 10px #d5d5d5;
```

Comment:
Not a huge deal, but it's a good practice to move colors to the variables and reuse those, especially if the same colors are used in multiple places